pkgs/data/icons/gruvbox-dark-icons-gtk/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-dark-icons-gtk";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmattheis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fks2rrrb62ybzn8gqan5swcgksrb579vk37bx4xpwkc552dz2z2";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/
+    rm README.md
+    cp -r ${src} $out/share/icons/${pname}
```

Do not copy `${src}` directly, because it is read only and it will be needed write  permission in order to generate the cache.

```suggestion
    cp -r * $out/share/icons/oomox-gruvbox-dark
    gtk-update-icon-cache $out/share/icons/oomox-gruvbox-dark
```

---

Is the dollar sign in `$out` not needed?